ceagle/api/v1/status.py

```diff
@@ -29,9 +32,17 @@ def get_status(period):
 
 
 @bp_status.route("/health/<period>")
-@fake_status.get_status_health
 def get_status_health(period):
-    return flask.jsonify("fixme!")
+    health_client = client.get_client("health")
+    if not health_client:
+        return flask.make_response(
+            json.dumps({"error": "No health endpoint configured"}),
+            500,
```

Comment:
500 does not mean "unexpected server error", but rather internal server error. https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html 

>    The server encountered an unexpected condition which prevented it
>    from fulfilling the request.
That's exactly what we have. The server is not configured properly to handle this request. 404 is not applicable here (again refer to the rfc):
> The server has not found anything matching the Request-URI
We've found all we have, but the server is misconfigured.

503 is definitely not applicable, see rfc
502 might be a better choice, but since the upstream is not configured at all — 500 is the best status here.